deeppavlov/configs/kbqa/kbqa_eng.json

```diff
@@ -0,0 +1,282 @@
+      },
+      {
+        "in": [
+          "x_emb"
+        ],
+        "in_y": [
+          "y_onehot"
+        ],
+        "out": [
+          "y_pred_probas"
+        ],
+        "main": true,
+        "class_name": "keras_classification_model",
+        "save_path": "{MODEL_PATH}/relation_prediction/model",
+        "load_path": "{MODEL_PATH}/relation_prediction/model",
+        "embedding_size": "#my_embedder.dim",
+        "n_classes": "#classes_vocab.len",
+        "kernel_sizes_cnn": [
+          3,
+          5,
+          7
+        ],
+        "filters_cnn": 256,
+        "optimizer": "Adam",
+        "learning_rate": 0.01,
+        "learning_rate_decay": 0.1,
+        "loss": "binary_crossentropy",
+        "last_layer_activation": "softmax",
+        "coef_reg_cnn": 1e-3,
+        "coef_reg_den": 1e-2,
+        "dropout_rate": 0.5,
+        "dense_size": 100,
+        "model_name": "cnn_model"
+      },
+      {
+        "in": "y_pred_probas",
+        "out": "y_pred_ids",
+        "class_name": "proba2labels",
+        "max_proba": true
+      },
+      {
+        "in": ["x"],
+        "class_name": "lazy_tokenizer",
+        "out": ["x_tokens"]
+      },
+      {
+        "in": ["x_tokens"],
+        "class_name": "str_lower",
+        "out": ["x_lower"]
+      },
+      {
+        "in": ["x_lower"],
+        "class_name": "sanitizer",
+        "nums": true,
+        "out": ["x_san"]
+      },
+      {
+        "in": ["y"],
+        "id": "tag_vocab",
+        "class_name": "simple_vocab",
+        "pad_with_zeros": true,
+        "fit_on": ["y"],
+        "save_path": "{MODEL_PATH}/ner_kb_eng/tag.dict",
+        "load_path": "{MODEL_PATH}/ner_kb_eng/tag.dict",
+        "out": ["y_ind"]
+      },
+      {
+        "in": ["x_tokens"],
+        "class_name": "char_splitter",
+        "out": ["x_char"]
+      },
+      {
+        "in": ["x_char"],
+        "id": "char_vocab",
+        "class_name": "simple_vocab",
+        "pad_with_zeros": true,
+        "fit_on": ["x_char"],
+        "save_path": "{MODEL_PATH}/ner_kb_eng/char.dict",
+        "load_path": "{MODEL_PATH}/ner_kb_eng/char.dict",
+        "out": ["x_char_ind"]
+      },
+      {
+        "in": ["x_tokens"],
+        "class_name": "mask",
+        "out": ["mask"]
+      },
+      {
+        "in": ["x_san"],
+        "id": "glove_emb",
+        "class_name": "glove",
+        "pad_zero": true,
+        "load_path": "{DOWNLOADS_PATH}/embeddings/glove.6B.100d.txt",
+
+        "out": ["x_emb"]
+      },
+      {
+        "id": "embeddings_char",
+        "class_name": "emb_mat_assembler",
+        "character_level": true,
+        "emb_dim": 32,
+        "embedder": "#glove_emb",
+        "vocab": "#char_vocab"
+      },
+      {
+        "id": "capitalization",
+        "class_name": "capitalization_featurizer",
+        "in": ["x_tokens"],
+        "out": ["cap"]
+      },
+      {
+        "in": ["x_emb", "mask", "x_char_ind", "cap"],
+        "in_y": ["y_ind"],
```

Comment:
Also, are you training NER and Classification models on the same expected outputs?

Reply:
No, outputs are different.